File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StudioPage.java
package com.example.owner_pc.animechecker.model.entity;

import java.util.ArrayList;

/**
 * Created by owner-PC on 2017/05/21.
 */

public class StudioPage extends Studio {
    public ArrayList<Anime> anime;

    @Override
    public String toString() {
        return "StudioPage{" +
                "anime=" + anime +
                '}';
    }
}

File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StudioPage.kt
package com.example.owner_pc.animechecker.model.entity

import java.util.*

/**
 * Created by owner-PC on 2017/05/21.
 */

class StudioPage : Studio() {
    var anime: ArrayList<Anime>? = null

    override fun toString(): String {
        return "StudioPage{" +
                "anime=" + anime +
                '}'
    }
}
